pkg/util/crypto.go

```diff
@@ -31,34 +32,103 @@ import (
 	"time"
 )
 
+func GenerateCACert(certPath, keyPath string) error {
```

Comment:
It might be cleaner if this returns the certificate and key, that way you could avoid re-parsing them from disk in the next call to GenerateSignedCert. What do you think?